File: Java/JavaDatabaseFrameworks/Common/src/main/java/org/dbFrameworks/Facade/IFrameworkTestsFacade.java
package org.dbFrameworks.Facade;

public interface IFrameworkTestsFacade {
    String RunAllTests(int samplesQuantity, String fileName);
    String SingleRecordSearch(int samplesQuantity);
    String SetOfDataSearch(int samplesQuantity);
    String SetOfDataWithIsNullSearch(int samplesQuantity);
    String AddRecords(int samplesQuantity);
    String EditRecords(int samplesQuantity);
    String DeleteRecords(int samplesQuantity);
}

File: C#/FrameworksPerformance/Common/Facades/IFrameworkTestsFacade.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Common.Facades
{
    public interface IFrameworkTestsFacade
    {
        string RunAllTests(int samplesQuantity, string fileName);
        string SingleRecordSearch(int samplesQuantity);
        string SetOfDataSearch(int samplesQuantity);
        string SetOfDataWithIsNullSearch(int samplesQuantity);
        string AddRecords(int samplesQuantity);
        string EditRecords(int samplesQuantity);
        string DeleteRecords(int samplesQuantity);
        string SearchTwoRelatedTables(int samplesQuantity);
    }
}
